tests/checkpointing/test_trainer_checkpoint.py

```diff
@@ -87,3 +93,49 @@ def test_accumulated_gradient_batches_with_ckpt_path(tmpdir):
     trainer_kwargs["max_epochs"] = 2
     trainer = Trainer(**trainer_kwargs)
     trainer.fit(model, ckpt_path=ckpt.last_model_path)
+
+
+def test_trainer_save_checkpoint_storage_options(tmpdir):
+    """This test validates that storage_options argument is properly passed to ``CheckpointIO``"""
+    model = BoringModel()
+    trainer = Trainer(
+        default_root_dir=tmpdir,
+        max_epochs=1,
+        limit_train_batches=1,
+        limit_val_batches=0,
```

Comment:
Would this work?

```suggestion
        fast_dev_run=True
```

Reply:
ah yes! the fast_dev_run skip saving check is only in `ModelCheckpoint` callback, not the actual `trainer.save_checkpoint` call